### ISSUE TITLE
Zef attempts to run .txt files in t/

### ISSUE THREAD

**Issue by AlexDaniel** [ugexe/zef#286]
```bash
zef install URI
```
```
===> Searching for: URI
===> Testing: URI:ver<.0.1.4>
===SORRY!=== Error while compiling /home/alex/.zef/store/uri.git/e5c85512db5446fa4f7a38e421170cd602a53dce/t/expected.txt
Confused
at /home/alex/.zef/store/uri.git/e5c85512db5446fa4f7a38e421170cd602a53dce/t/expected.txt:1
------> file:⏏///space/pub/music/mp3/Musopen DVD/Brahm
    expecting any of:
        colon pair
===> Testing [FAIL]: URI:ver<.0.1.4>
Aborting due to test failure: URI:ver<.0.1.4> (use --force-test to override)
```

```bash
perl6 --version
```
```
This is Rakudo version 2018.12 built on MoarVM version 2018.12
implementing Perl 6.d.
```
```bash
zef --version
```
```
Version detection requires rakudo v2019.01 or newer
```

**Comment by ugexe** [comment]
Resolved in e1da41a4a6bfd5ac68f78b930fe6c090e182efbb

**Comment by AlexDaniel** [comment]
That was quick! Thank you, @ugexe!
